jdaviz/configs/imviz/tests/test_links_control.py

```diff
@@ -0,0 +1,13 @@
+from jdaviz.configs.imviz.tests.utils import BaseImviz_WCS_NoWCS
+
+
+class TestLinksControl(BaseImviz_WCS_NoWCS):

```

Comment:
If you want to test linking by WCS, please use the `BaseImviz_WCS_WCS` test class. This one will just silently falls back to pixel linking.

Reply:
ah, didn't realize the difference.  The main intent of the test was to force coverage and the default of the "fast approximation" (linking itself is covered differently), but might as well change it to make sure the right linking is being done, so thanks!